core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-master.yml

```diff
@@ -60,7 +60,8 @@
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl drain {{ inventory_hostname }} --ignore-daemonsets --delete-local-data
   when:
-    - groups['kubernetes_node'] is defined and groups['kubernetes_node']|list|length > 0 # drain master only if there is at least one worker node
+    - groups['kubernetes_node'] is defined
+    - groups['kubernetes_node']|length > 0 # drain master only if there is at least one worker node
```

Comment:
```suggestion
    - groups['kubernetes_node']|list|length > 0 # drain master only if there is at least one worker node
```

Reply:
In this case casting to list is not needed (tested)